package.json

```diff
@@ -84,6 +84,7 @@
     "react-sortable-hoc": "^0.6.7",
     "redux": "^3.5.2",
     "sander": "^0.5.1",
+    "sanitize-html": "^1.18.2",
```

Comment:
Hi @Redsandro 

yarn.lock was not updated, could you please update to keep everyone in sync? 

Thanks

Reply:
@rayou this branch was merged 2 days ago. You'll have to create a PR upstream.

Please note that I am using the `npm` workflow. I don't know what `yarn` is. I just `git push` what was updated. If this creates a mismatch, perhaps some automation should be written.